src/sentry/static/sentry/app/views/admin/adminWarnings.tsx

```diff
@@ -3,13 +3,16 @@ import React from 'react';
 import AsyncView from 'app/views/asyncView';
 import {t} from 'app/locale';
 
-export default class AdminSettings extends AsyncView {
-  getEndpoints() {
-    return [['data', this.getEndpoint()]];
-  }
+type Data = {
+  groups: [groupName: string, grouppedWarnings: string[]][];
+  warnings: string[];
+};
+
+type State = AsyncView['state'] & {data: Data};
```

Comment:
Afaiu `AsyncComponent` sets these fetched values to `null` if there is an error, so we would want to account for that. Please correct me if I'm wrong @priscilawebdev 
```suggestion
type State = AsyncView['state'] & {data: Data | null};
```

Reply:
that is right @taylangocmen 😉

Reply:
I give in 🤐